electron-start.html

```diff
@@ -2,15 +2,34 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <style type="text/css">
```

Comment:
I should probably move this into it's own css file. It might even make sense to move all of the electron-specific stuff into it's own sub-folder instead of cluttering up the root of the repo. I'd like to hear the opinion of maintainers about how the electron wrapper should be organized relative to the web app.